### PR TITLE
fix wait-for-it script

### PR DESCRIPTION
**Pull request by marten-seemann**
`net.DialUDP` gives a connected UPD socket, i.e. one that will read ICMP Unreachable packets. That's the opposite of what we want: we want to periodically send out a packet, and then just read from the connection to see when we get a Version Negotiation packet as a response.

### CODE REVIEW

File: sim/wait-for-it-quic/wait-for-it.go [hunk 1]
```diff
@@ -32,39 +32,54 @@ func main() {
...
+		}
+		if !bytes.Equal(b[1:5], []byte{0, 0, 0, 0}) {
+			log.Fatal("expected Version Negotiation packet")
+		}
+		close(done)
+	}()
+
+	ticker := time.NewTicker(time.Second / 2)
+	if timeout == 0 {
+		timeout = 365 * 24 * time.Hour
```
Comment (janaiyengar) [inline sim/wait-for-it-quic/wait-for-it.go:69]: What is the value of this timeout?

**Review by janaiyengar — APPROVED** [review]
LGTM -- modulo the `go` syntax. And one question inline.

File: sim/wait-for-it-quic/wait-for-it.go (hunk 1 above)
Comment (marten-seemann) [reply sim/wait-for-it-quic/wait-for-it.go:69]: Using a practically infinite value it no `-t` is given.

Comment (marten-seemann) [reply sim/wait-for-it-quic/wait-for-it.go:69]: You're right, it's ugly. Will fix.

**Pull request merged**